lib/cooked_processor_mixin.rb

```diff
@@ -251,6 +253,33 @@ def add_broken_image_placeholder!(img)
     true
   end
 
+  def add_blocked_hotlinked_image_placeholder!(el)
+    el.name = "a"
+    el.set_attribute("href", el[PrettyText::BLOCKED_HOTLINKED_SRC_ATTR])
+    el.set_attribute("target", "_blank")
+    el.set_attribute("class", "blocked-hotlinked-placeholder")
+    el.set_attribute("title", I18n.t("post.image_placeholder.blocked_hotlinked_title"))
+    el << "<svg class=\"fa d-icon d-icon-link svg-icon\" aria-hidden=\"true\"><use href=\"#link\"></use></svg>"
+    el << "<span class=\"notice\">#{CGI.escapeHTML(I18n.t("post.image_placeholder.blocked_hotlinked"))}</span>"
+
+    true
+  end
+
+  def add_blocked_hotlinked_media_placeholder!(el, src)
+    placeholder = Nokogiri::XML::Node.new("a", el.document)
+    placeholder.name = "a"
+    placeholder.set_attribute("href", src)
+    placeholder.set_attribute("target", "_blank")
```

Comment:
Same - should it also set `rel="noopener noreferrer"`?

Reply:
Good call! Looks like it's automatically added by a later step in the CookedPostProcessor, so no need to add anything specific here. 

Added a safety-check in the specs to make sure that doesn't regress: 6ca0eb136e70a7a4484af2a732aa4f9aed160fa9